arangod/Aql/AqlFunctionFeature.cpp

```diff
@@ -435,6 +435,10 @@ void AqlFunctionFeature::addMiscFunctions() {
   add({"DECODE_REV", ".", flags, &Functions::DecodeRev});
   add({"V8", ".", Function::makeFlags(FF::Deterministic, FF::Cacheable)});  // only function without a
                                                                             // C++ implementation
+                                                                            //
+  auto validationFlags = Function::makeFlags(FF::None);
+  add({"GET_SCHEMA", ".", validationFlags, &Functions::GetSchema});
+  add({"SCHEMA_VALIDATE", ".,.", validationFlags, &Functions::SchemaValidate});
```

Comment:
The naming feels consistent. Wouldn't `SCHEMA_GET` and `SCHEMA_VALIDATE` (or the other way around) be better than placing the verb at the beginning in the former and at the end in the latter case?

Reply:
Good point. However, the current list of functions does not indicate a clear preference about whether the verb should go first or not:
``` 
"IS_NULL"
"IS_BOOL"
"IS_NUMBER"
"IS_STRING"
"IS_ARRAY"
"IS_OBJECT"
"IS_DATESTRING"
"IS_KEY"
"TO_NUMBER"
"TO_STRING"
"TO_BOOL"
"TO_ARRAY"
"CONCAT_SEPARATOR"
"CHAR_LENGTH"
"REGEX_MATCHES"
"REGEX_SPLIT"
"REGEX_TEST"
"REGEX_REPLACE"
"FIND_FIRST"
"FIND_LAST"
"TO_BASE64"
"TO_HEX"
"ENCODE_URI_COMPONENT"
"LEVENSHTEIN_DISTANCE"
"LEVENSHTEIN_MATCH"
"NGRAM_MATCH"
"NGRAM_SIMILARITY"
"NGRAM_POSITIONAL_SIMILARITY"
"IN_RANGE"
"RANDOM_TOKEN"
"UNION_DISTINCT"
"VARIANCE_SAMPLE"
"VARIANCE_POPULATION"
"STDDEV_SAMPLE"
"STDDEV_POPULATION"
"COUNT_DISTINCT"
"SORTED_UNIQUE"
"REMOVE_VALUE"
"REMOVE_VALUES"
"REMOVE_NTH"
"REPLACE_NTH"
"MERGE_RECURSIVE"
"UNSET_RECURSIVE"
"JSON_STRINGIFY"
"JSON_PARSE"
"IS_IN_POLYGON"
"GEO_DISTANCE"
"GEO_CONTAINS"
"GEO_INTERSECTS"
"GEO_EQUALS"
"GEO_AREA"
"GEO_POINT"
"GEO_MULTIPOINT"
"GEO_POLYGON"
"GEO_MULTIPOLYGON"
"GEO_LINESTRING"
"GEO_MULTILINESTRING"
"DATE_TIMESTAMP"
"DATE_ISO8601"
"DATE_DAYOFWEEK"
"DATE_YEAR"
"DATE_MONTH"
"DATE_DAY"
"DATE_HOUR"
"DATE_MINUTE"
"DATE_SECOND"
"DATE_MILLISECOND"
"DATE_DAYOFYEAR"
"DATE_ISOWEEK"
"DATE_LEAPYEAR"
"DATE_QUARTER"
"DATE_DAYS_IN_MONTH"
"DATE_ADD"
"DATE_SUBTRACT"
"DATE_DIFF"
"DATE_COMPARE"
"DATE_FORMAT"
"DATE_TRUNC"
"DATE_ROUND"
"DATE_NOW"
"NOT_NULL"
"FIRST_LIST"
"FIRST_DOCUMENT"
"PARSE_IDENTIFIER"
"IS_SAME_COLLECTION"
"DECODE_REV"
"CURRENT_USER"
"CURRENT_DATABASE"
"CHECK_DOCUMENT"
"COLLECTION_COUNT"
"PREGEL_RESULT"
"WITHIN_RECTANGLE"
```

Reply:
https://github.com/arangodb/arangodb/pull/11327#discussion_r397036350 not my idea! I would have preferred `GET_VALIDATION` and `VALIDATE` but that is history and we are stuck with JSON Schema. Which is a shitty format for many reasons.

Reply:
It is not very helpful to complain about this decision at all occasions.
The question here is whether we named it `GET_SCHEMA` and `VALIDATE_SCHEMA` or if we name it `SCHEMA_GET` and `SCHEMA_VALIDATE`.
No need to always restart the discussion about JSON schema in general.

Reply:
I have probably worked more than anyone else with the format and I think it is sad that nobody will listen. I have seen other formats that provide far more flexibility like being conditional in case of presence or absence of attributes. It is immensely frustrating that stuff like that is not taken into account.

Reply:
Yes. But please now accept the decision we took.
It is not helpful to always restart this discussion. It will not help us, and I want to move on with things.
@Simran-B : what shall we do regarding the naming of the new functions? We do not seem to have real consistency with the existing functions' naming, so there is no clear example.

Reply:
While there is no strict convention, I would prefer `SCHEMA_GET` and `SCHEMA_VALIDATE` so that they are grouped together by name (the docs are alphabetically sorted for instance). We do this for `GEO_*()`, `DATE_*()` and many other functions already.

Reply:
@Simran-B : I think this is a good idea and is definitely an improvement. 
@ObiWahn : can we please adjust the function names according to Simran's suggestion?